runners/spark/pom.xml

```diff
@@ -147,6 +147,29 @@
       <scope>provided</scope>
     </dependency>
     <dependency>
+      <groupId>com.esotericsoftware.kryo</groupId>
+      <artifactId>kryo</artifactId>
+      <version>2.21</version>
+      <scope>provided</scope>
+    </dependency>
+    <dependency>
+      <groupId>de.javakaffee</groupId>
+      <artifactId>kryo-serializers</artifactId>
+      <version>0.39</version>
```

Comment:
I went with `0.39` because I added `ReverseList` support there while working on Spark 2 (depends on `chill` 0.8.0 which depends on `kryo` 3.0.3).
I'll update this to match the `kryo` version for Spark 1.6.x (`chill` 0.5.0) if not backward compatible.
Let me check that..